spec/hosts/roles/rke_spec.rb

```diff
@@ -35,7 +35,7 @@
     )
   end
 
-  if (site == 'dev') || (site == 'tu')
+  if (site == 'dev') || (site == 'tu') || (site == 'ls')
```

Comment:
This should probably be a case statement but I realize I'm the one that started if off with an if/or.

Reply:
great suggestion, changed.